source/install/upgrade_from_34.rst

```diff
@@ -41,6 +44,11 @@ Upgrading on Debian or Ubuntu
 
     sudo apt-get remove percona-server-mongodb-34*
 
+#. Optionally add ``useDeprecatedMongoRocks: true`` line into the config
+   file if using MongoRocks storage engine::
+
+    sed -i '/engine: *rocksdb/a\ useDeprecatedMongoRocks: true' /etc/mongod.conf
```

Comment:
`sed -i '/engine:[ \t]*rocksdb/a\ useDeprecatedMongoRocks: true' /etc/mongod.conf`

Is safer as it includes tabs besides spaces